terraform/modules/redis/main.tf

```diff
@@ -0,0 +1,54 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.0"
+    }
+  }
+}
+
+/// elasticache cluster
+
+resource "aws_elasticache_subnet_group" "redis_cluster_subnet_group" {
+  name       = var.cluster_name
+  subnet_ids = var.subnet_ids
+}
+
+resource "aws_security_group" "redis" {
+  name        = "${var.cluster_name}_elasticache"
+  vpc_id      = var.vpc_id
+  description = "${var.cluster_name} elasticache cluster"
```

Comment:
This description doesn't add anything - consider removing.

Reply:
I will remove